### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by faraznaeem**
## Add Readme to gigafood ##

### CODE REVIEW

**Comment by tochman** [comment]
Pull from upstream, run `bundle` and push to this branch to fix the failing tests @faraznaeem

**Comment by faraznaeem** [comment]
getting merge conflicts, can I safely remove the gemfile.lock and then pull from upstream?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-# gigafood
-CraftAcademy - Summer 2017 Final Project
+# Gigafood
+
+GigaFood is a catering businessthat creates organic plant-based vegan dishes that are marvelous to eat, high in nutrition, good for your health and sustainable for the planet.
```
Comment (tochman) [inline README.md]: Typo. ;-)

**Review by tochman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (faraznaeem) [reply README.md]: Done :)

**Review by tochman — APPROVED** [review]

**Pull request merged**